### DISCUSSION TITLE
Strange behaviour with Metdata Processor

### DISCUSSION THREAD

**Discussion by ElyumOfTheRocks** [UMEP-dev/UMEP#420] [Q&A] [answered]
Hi!

I am trying to convert .epw files with the metdata processor and I was wondering if my results were normal. Whathever the .epw file I try to convert, the first column (the year) get set to 1985. See .txt attached.
[010716_yul.txt](https://github.com/UMEP-dev/UMEP/files/8781475/010716_yul.txt)

For reference, here is the .epw file I used (converted in .txt because gh dosen't support .epw)
[010716_yul_epw.txt](https://github.com/UMEP-dev/UMEP/files/8781487/010716_yul_epw.txt)

I find this really strange, but I saw the same behaviour in that tutorial video, hence my doubt. https://youtu.be/bavT3RFvusE?t=415

Thanks for your time

_win10
QGIS 3.22.6
UMEP 3.19.3_

**Comment by biglimp** [comment, marked as answer]
This is because an epw-file can include data from different years e.g. January is from 1996, February from 2003 etc. Some models in UMEP requires continouos data to run, hence one common year is used, in this case 1985. This is hard coded into the tool. If you like any other year, just change manually after the conversion.

**Reply by ElyumOfTheRocks** [reply]
I understand better. Thanks for the prompt reply! Love your work
